### ISSUE TITLE
Fix packaging warnings for `probe-plugin`

### ISSUE THREAD

**Issue by LukaszKontowski** [VirtusLab/ide-probe#243]
In compilation, following warnings appear:
```
[warn] Warning: {api_2_12} will be merged into non-terminal {probe-plugin_2_12}, other candidates were: Set({pants-probe-plugin_2_12}, {bazel-probe-plugin_2_12}, {scala-probe-plugin_2_12})
[warn] You can specify explicit merge: packageMethod := PackagingMethod.MergeIntoOther(pants-probe-plugin_2_12)
[warn] Warning: {api_2_13} will be merged into non-terminal {probe-plugin_2_13}, other candidates were: Set({pants-probe-plugin_2_13}, {scala-probe-plugin_2_13}, {bazel-probe-plugin_2_13})
[warn] You can specify explicit merge: packageMethod := PackagingMethod.MergeIntoOther(pants-probe-plugin_2_13)
[warn] Warning: {api_2_12} will be merged into non-terminal {probe-plugin_2_12}, other candidates were: Set({pants-probe-plugin_2_12}, {bazel-probe-plugin_2_12}, {scala-probe-plugin_2_12})
[warn] You can specify explicit merge: packageMethod := PackagingMethod.MergeIntoOther(pants-probe-plugin_2_12)
[warn] Warning: {api_2_13} will be merged into non-terminal {probe-plugin_2_13}, other candidates were: Set({pants-probe-plugin_2_13}, {scala-probe-plugin_2_13}, {bazel-probe-plugin_2_13})
[warn] You can specify explicit merge: packageMethod := PackagingMethod.MergeIntoOther(pants-probe-plugin_2_13)
```
The reason of this issue is to fix this.

**Comment by PawelLipski** [comment]
`-Xfatal-warnings` to be added to flags once this is fixed

**Comment by LukaszKontowski** [comment]
These warnings come from the `sbt-idea-plugin`:
https://github.com/JetBrains/sbt-idea-plugin/blob/master/packaging/src/main/scala/org/jetbrains/sbtidea/packaging/mappings/LinearMappingsBuilder.scala#L86-L87

**Comment by LukaszKontowski** [comment]
Waiting for sbt-idea-plugin team to respond: https://github.com/JetBrains/sbt-idea-plugin/issues/118

**Comment by PawelLipski** [comment]
So to recap... `-Xfatal-warnings` can be added regardless? let's assume the answer on the issue in sbt-idea-plugin won't come anytime soon

**Comment by LukaszKontowski** [comment]
Yes - PR created https://github.com/VirtusLab/ide-probe/pull/247

**Comment by PawelLipski** [comment]
Unassigning Łukasz as this is blocked on JetBrains, dunno when they're gonna respond :/
